packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -47,9 +55,13 @@ export class QuickOpenWorkspace {
         await this.preferences.ready;
         if (!workspaces.length) {
             this.items.push({
-                label: 'No Recent Workspaces'
+                label: nls.localize('theia/windowActions/noRecentWorkSpaces', 'No Recent Workspaces')
             });
```

Comment:
We can potentially remove this item completely like in vscode if you prefer.

Reply:
I agree, it's best to have all strings on this class localized as per the current solution.
Done!